pandas/tests/frame/test_rank.py

```diff
@@ -310,6 +310,7 @@ def test_rank_pct_true(self, method, exp):
         tm.assert_frame_equal(result, expected)
 
     @pytest.mark.single
+    @pytest.mark.high_memory
```

Comment:
its fine to mark, but how is this skipped on 32 bit systems?

Reply:
@scarabeusiv did you determine that the issue was with the low memory of the machine? That it would have happened on a 64-bit system with the same amount of RAM?

Reply:
Yep with the suspicion it would happen on the same machine with the same amount of memory.

With the patch it is building fine:
https://build.opensuse.org/package/show/devel:languages:python:numeric/python-pandas

Unrelated: do you guys care about ppc/s390 build issues? :)

Reply:
We would take patches certainly. It's a bit hard to "officially" support a platform when it's not tested regularly as part of our CI though.

@jreback does that answer your question about skipping?

Reply:
we should skip skip these on 32-bit (eg a conditional skip in pd.test() but can be followip

Reply:
Why do we need to skip on 32 bit?

Reply:
> Yep with the suspicion it would happen on the same machine with the same amount of memory.

so the issue is the available RAM, not whether the OS is 32 or 64 bit.